site/gatsby-site/src/components/discover/REFINEMENT_LISTS.js

```diff
@@ -92,6 +92,15 @@ const REFINEMENT_LISTS = [
     faClasses: 'far fa-flag',
     type: 'refinement',
   },
+  {
+    attribute: 'is_incident_report',
+    placeholder: 'none',
+    label: 'dsad',
```

Comment:
Change `dsad` label to something meaningful. It's also visible on mobile:

![image](https://user-images.githubusercontent.com/6564809/199126379-bb32434b-4094-41d5-82f7-3e6645f81ecb.png)

Reply:
Yeah `dasd` it is because it shouldn't be visible, it's hidden now 👍